src/cswinrt/code_writers.h

```diff
@@ -1427,7 +1427,9 @@ remove => %.% -= value;
                 }
             }
             // Skip metadata attributes
-            if (attribute_namespace == "Windows.Foundation.Metadata" && attribute_name != "DefaultOverload" && attribute_name != "Overload") continue;
+            if (attribute_namespace == "Windows.Foundation.Metadata" && 
```

Comment:
What do you think of moving the check to a predicate and calling that the function from the `if ()` ?

Reply:
yes, that kind of factoring could be helpful if called multiple times.  here, probably just adds unnecessary complexity.